core/src/main/scala/com/banno/kafka/consumer/ConsumerOps.scala

```diff
@@ -473,6 +489,12 @@ case class ConsumerOps[F[_], K, V](consumer: ConsumerApi[F, K, V]) {
 
     def reset(time: FiniteDuration): OffsetCommitState =
       OffsetCommitState.empty(time)
+
+    def resetCount(time: FiniteDuration): OffsetCommitState =
```

Comment:
It seems like it's safe to reset only the count upon updating the ref. Previously, both the count and the offsets were zeroed out, and IIUC, since it was previously only events that caused offset committing, the zeroing out may have saved offsets from being redundantly committed, but semantically, it didn't matter either way. Please correct me if I misunderstand.

Reply:
That sounds correct: all offsets in this state are from successfully processed records, so we should always be able to commit them, even if they are redundant. Apparently it's that redundancy that we need so that Kafka does not delete committed offsets for low-volume topics.